test/test.js

```diff
@@ -66,6 +66,10 @@ test('override negations and re-include explicitly defined patterns', t => {
 	t.deepEqual(multimatch(['foo', 'bar', 'baz'], ['!*a*', '*z']), ['baz']);
 });
 
+test('list be sorted when order enabled', t => {
+	t.deepEqual(multimatch(['foo', 'bar', 'baz'], ['*', '!*{o,r}', 'foo'], {ordered: true}), ['foo', 'baz']);
+});
```

Comment:
This will need more tests. For example, the case mentioned in the issue: https://github.com/sindresorhus/multimatch/issues/14